pkg/api/v1/clients/kube/controller/controller.go

```diff
@@ -138,7 +138,7 @@ func (c *Controller) processNextWorkItem() bool {
 		}
 		switch e.eventType {
 		case added:
-			c.handler.OnAdd(e.new)
```

Comment:
Note: this was the only manual change to the go code.

Reply:
Interesting change! https://github.com/kubernetes/kubernetes/issues/113763